bluesky/run_engine.py

```diff
@@ -703,28 +673,11 @@ def request_pause(self, defer=False):
         self._deferred_pause_requested = False
         self._interrupted = True
         self._state = 'pausing'
-
-        self._record_interruption('pause')
+        for run in self._run_bundlers.values():
+            run._record_interruption('pause')
```

Comment:
Should we make all the API on the Bundler than needs to be called by the RunEngine public?

Reply:
I could go either way on this, on one hand that seems like the right Python thing to do, on the other hand, Bundler it self is a very internal thing and I don't think we want users to be actively poking at them (which is what public methods suggest you can do!).

I think the RE touching private methods on Bundler is ok based on an analogy to `friend` classes in c++ .

Reply:
These were made public.